photon-client/src/main/scala/com/linkedin/photon/ml/data/avro/AvroUtils.scala

```diff
@@ -96,8 +96,8 @@ object AvroUtils {
    * @param outputDir The output directory to save the data as Avro files
    * @param schemaString The schema string of the data
    */
-  protected[ml] def saveAsAvro[T <: SpecificRecord : ClassTag](data: RDD[T], outputDir: String, schemaString: String): Unit = {
-    val job = new JobConf
+  protected[ml] def saveAsAvro[T <: SpecificRecord : ClassTag](data: RDD[T], outputDir: String, schemaString: String, jobConf: Option[JobConf] = None): Unit = {
```

Comment:
This doesn't need to be an `Option` -- we can just default to `new JobConf` in the signature, e.g.:

```scala
protected[ml] def saveAsAvro[T <: SpecificRecord : ClassTag](
    data: RDD[T],
    outputDir: String,
    schemaString: String,
    jobConf: JobConf = new JobConf): Unit = {
```

Reply:
done